include/drivers/gpio.h

```diff
@@ -817,13 +823,15 @@ static inline int gpio_port_set_clr_bits_raw(struct device *port,
 /**
  * @brief Set logical level of selected output pins.
  *
- * Pins which are simultaneously selected by set_pins and clear_pins value will
- * be toggled.
+ * Note that the behavior required for pins present in both
+ * ``set_pins`` and ``clear_pins`` is to toggle the current pin
+ * setting.
```

Comment:
This text is written as if it was an instruction for a person implementing the driver. The API documentation should be targeted at the users of the API.